Configurations/15-ios.conf

```diff
@@ -1,9 +1,8 @@
 #### iPhoneOS/iOS
 #
-# It takes recent enough Xcode to use following two targets. It shouldn't
-# be a problem by now, but if they don't work, original targets below
-# that depend on manual definition of environment variables should still
-# work...
+# Targets that ends with `xcrun` requires a recent enough Xcode which supports command-line support to determine the
+# applicable c-compiler through `xcrun -sdk iphoneos`. If you build machine does not support this command then use the
+# legacy targets in the end of this file that depend on manual definition of environment variables.
```

Comment:
A number of minor issues with this paragraph: longer lines (typically around 80 chars is preferred); "targets" should have "require" not "requires" (singular/plural); should be "your build machine" not "you build machine', but might be better in third person anyway.

How about

```
# `xcrun` targets require an Xcode that can determine the correct C compiler via
# `xcrun -sdk iphoneos`. This has been standard in Xcode for a while now - any recent
# Xcode should do.  If the Xcode on the build machine doesn't support this then use
# the legacy targets at the end of this file. These require manual definition of
# environment variables.
```

Reply:
Good suggestion!